tasks/task07/order_repository.py

```diff
@@ -39,8 +39,8 @@ def __eq__(self, o: object) -> bool:
         return False
 
 class OrderRepository:
-    def __init__(self):
```

Comment:
add `.gitignore` file to the root of the repository and ignore `__pycache__` folder:

`exadel-python-course-2021/.gitignore`:
```
__pycache__
```